chia/wallet/trade_manager.py

```diff
@@ -230,12 +234,10 @@ async def cancel_pending_offer_safely(
             if wallet is None:
                 continue
 
-            if wallet.type() == WalletType.NFT:
-                new_ph = await wallet.wallet_state_manager.main_wallet.get_new_puzzlehash()
-            else:
-                new_ph = await wallet.get_new_puzzlehash()
+            # All the wallets currently return the standard wallet puz hash
```

Comment:
Looks like DataLayer still has a custom implementation.  Might do the same thing, didn't check yet.

https://github.com/Chia-Network/chia-blockchain/blob/060cf7a90bc6f293368df5647fe10b721ba8238e/chia/data_layer/data_layer_wallet.py#L1076-L1077

Reply:
yea, still looking into this - but quex thinks it's the same - but needs confirmation

Reply:
If confirmed the same, will it be removed?